src/main/resources/assets/computercraft/lua/rom/apis/parallel.lua

```diff
@@ -55,12 +87,21 @@ local function runUntilLimit( _routines, _limit )
     end
 end
 
+function createBundle( ... )
+    local bundle = {}
```

Comment:
Honestly, probably better to do something like this:

```lua
local bundle = setmetatable({
  run = true,
  coroutines = {},
}, blah)
```

Having `bundle` be a list, something to store data and something with methods is a odd.